config/routes.rb

```diff
@@ -20,4 +18,7 @@
   get '/pets/:id/edit', to: 'pets#edit'
   patch '/pets/:id', to: 'pets#update'
   delete '/pets/:id', to: 'pets#destroy'
+
+  get '/shelters/:shelter_id/reviews/new', to: 'reviews#new'
+  post '/shelters/:shelter_id', to: 'reviews#create'
```

Comment:
Can you explain this to me real quick? How/why does post `/shelters/:shelter_id` lead to reviews create?
Also, is there supposed to be a shelters review home page?

Reply:
We have to be able to leave a review for a specific shelter, which can only be done from that shelter's page, which is why we have to have /shelters/:shelter_id as the prefix. Is there another route you had in mind? 

And there was nothing about a review home page in the user story. The review is essentially just an edit to the shelter show page, but it has it's own data table. Everything written currently satisfies the user story.

Reply:
No route in mind, just needed to clear up my understanding. Thanks for the clarification, I actually just couldn't understand what the user story was asking for and I need to work on understanding routes.